common/components/controllers/EditProfileController.jsx

```diff
@@ -5,6 +5,7 @@ import DjangoCSRFToken from 'django-react-csrftoken'
 import UserAPIUtils from "../utils/UserAPIUtils.js";
 import type {UserAPIData} from "../utils/UserAPIUtils.js";
 import {CountrySelector, defaultCountryCode} from "../common/selection/CountrySelector.jsx";
```

Comment:
```
WARNING in ./common/components/controllers/EditProfileController.jsx 64:15-33
"export 'defaultCountryCode' was not found in '../common/selection/CountrySelector.jsx'
```

Webpack gives this warning on build for me. It might not be new?